app/Resources/views/macros/forms.html.twig

```diff
@@ -81,7 +81,7 @@
             <span class="text-muted">({{ msg("optional") }})</span>
         {% endif %}
     </label>
-    <input type="date" class="form-control" name="{{ type }}" placeholder="2016-11-01"{% if required %} required{% endif %} />
+    <input type="date" class="form-control" name="{{ type }}" placeholder="YYYY-MM-DD"{% if required %} required{% endif %} />
```

Comment:
What I'd really like to do is have it display the actual date that will be submitted... however; I'm not sure that's possible.